### DISCUSSION TITLE
Local absolute path is somehow saved in Primary config directory

### DISCUSSION THREAD

**Discussion by CytoShimon** [facebookresearch/hydra#2581] [Q&A]
Hi

sending a job to remote VM with the following code fails due to absolute path that is somehow saved. I did not find the absolute path in any attribute of ConfigStore or a place where it is saved (obviously not hard coded). Why is the path not taken during runtime in the VM?
How can I fix it?

Thanks
```

def run_experiment():

    from dis25.utils_config import Mainconfig
    from hydra.core.config_store import ConfigStore
    import hydra

    @hydra.main(version_base=None, config_path="conf", config_name="main_config")
    def _run(cfg: Mainconfig) -> None:

   CODE


    # define experiment config type
    cs = ConfigStore.instance()
    cs.store(name="run_experiment_config", node=Mainconfig)
    _run()

    return
```

**Comment by Jasha10** [comment]
Could you please post a minimal reproducible example of the problem you're seeing?

https://stackoverflow.com/help/minimal-reproducible-example
